server/sonar-web/src/main/js/apps/background-tasks/components/Workers.tsx

```diff
@@ -64,16 +66,30 @@ export default class Workers extends React.PureComponent {
     });
   };
 
-  closeForm = (newWorkerCount /*: ?number */) =>
+  closeForm = (newWorkerCount?: number) =>
     newWorkerCount
       ? this.setState({ formOpen: false, workerCount: newWorkerCount })
       : this.setState({ formOpen: false });
 
-  handleChangeClick = (event /*: Event */) => {
+  handleChangeClick = (event: React.SyntheticEvent<HTMLAnchorElement>) => {
     event.preventDefault();
     this.setState({ formOpen: true });
   };
 
+  handleHelpClick = (event: React.SyntheticEvent<HTMLAnchorElement>) => {
+    event.preventDefault();
+    event.stopPropagation();
+    this.toggleNoSupportPopup();
+  };
+
+  toggleNoSupportPopup = (show?: boolean) => {
+    if (show != undefined) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') != should be !== [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1440)